src/app/pages/SearchResultPage/searchResultPage.tsx

```diff
@@ -1,17 +1,50 @@
 import styles from "./searchResultPage.module.css";
+import useSet from "../../utils/useSet";
+import { Link, useNavigate } from "react-router-dom";
+import usePostSet from "../../utils/usePostSet";
+import { FormEvent } from "react";
+
+export default function SearchResultPage(): JSX.Element {
+  const query = localStorage.getItem("Search Query");
+  const navigate = useNavigate();
+  const { searchResult } = useSet(query);
+
+  const numberSet = searchResult?.numberSet;
+  const nameSet = searchResult?.nameSet;
+  const numberParts = searchResult?.numberParts;
+  const year = searchResult?.year;
+  const imageUrl = searchResult?.imageUrl;
+  const nameTheme = searchResult?.nameTheme;
```

Comment:
Not 100 percent sure, but i believe u can use destructuring assignment like  "const {numberSet, nameSet ... } = serchResult"